src/xmpp/iq.c

```diff
@@ -1612,35 +1612,54 @@ _version_get_handler(xmpp_stanza_t* const stanza)
     xmpp_ctx_t* const ctx = connection_get_ctx();
     const char* id = xmpp_stanza_get_id(stanza);
     const char* from = xmpp_stanza_get_from(stanza);
+    ProfAccount* account = accounts_get_account(session_get_account_name());
+    GString* client = account->client != NULL ? g_string_new(account->client) : NULL;
+    bool is_custom_client = client != NULL;
+    gchar* custom_version_str = NULL;
+    if (is_custom_client) {
+        custom_version_str = strstr(client->str, " ");
+        if (custom_version_str != NULL) {
+            *custom_version_str = '\0'; // Split string on name and version
+            custom_version_str++;
+        }
+        cons_show("Received IQ version request (XEP-0092) from %s. Client: %s; ver: %s", from, client->str, custom_version_str);
+    }
```

Comment:
```suggestion
    auto_char char* client = account->client != NULL ? strdup(account->client) : NULL;
    bool is_custom_client = client != NULL;
    gchar* custom_version_str = NULL;
    if (is_custom_client) {
        custom_version_str = strstr(client, " ");
        if (custom_version_str != NULL) {
            *custom_version_str = '\0'; // Split string on name and version
            custom_version_str++;
        }
        cons_show("Received IQ version request (XEP-0092) from %s. Client: %s; ver: %s", from, client, custom_version_str);
    }
```

We're moving away from using `GString` since its overhead is really not required in most cases. This is one of those were a simply `strdup()` is sufficient. If you use `auto_char` the compiler will take care of `free()`'ing that newly allocated string.